altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,7773 @@
+# The contents of this file are automatically written by
+# tools/generate_schema_wrapper.py. Do not modify directly.
+
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING, Any, Literal, Sequence, TypedDict
+
+if sys.version_info >= (3, 14):
```

Comment:
Maybe already on your radar with #3587. This probably requires a change in `pyproject.toml` so that `typing_extensions` is already installed for Python 3.13? https://github.com/vega/altair/blob/bf29786819a4287de83adfe034d927d1cc208ac8/pyproject.toml#L18

Reply:
> Maybe already on your radar with #3587. This probably requires a change in `pyproject.toml` so that `typing_extensions` is already installed for Python 3.13?

Yeah definitely on my radar, I'd already made the change in https://github.com/vega/altair/pull/3591/files

That PR will be blocked until https://github.com/geopandas/pyogrio/issues/470 is resolved though

@binste what do you think about splitting that change into another PR that we can merge quickly?

I'm thinking it would be helpful to have a dependency change in our release notes; even though it is technically just "unbreaking" `3.13`

Reply:
Resolved in https://github.com/vega/altair/pull/3593